sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/GrantAccessData.cs

```diff
@@ -36,15 +36,11 @@ public GrantAccessData()
         /// SAS access expires.</param>
         /// <param name="getSecureVMGuestStateSAS">Set this flag to true to get
         /// additional SAS for VM guest state</param>
-        /// <param name="fileFormat">Used to specify the file format when
```

Comment:
Removing `fileFormat` is a breaking change. You can add back the constructor and `FileFormat` property, but make it useless in reality.